scripts/ort-evaluator.sh

```diff
@@ -4,13 +4,16 @@
 # against customizable policy rules.
 
 if [[ "$ORT_DISABLE_SCANNER" = "true" ]]; then
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ANALYZER_FILE
-elif [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ADVISOR_FILE
+    if [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
```

Comment:
Nit: Let's stick to the same logic as one line above and compare with `= "true"`, swapping the code branches below.

Reply:
I guess something went wrong while addressing this: The change per se is fine with me and should stay. Just the order of the introduced branches in the nested condition should be swapped, to match the logic of the surrounding condition.

Reply:
next try :) I hope this is fine now

Reply:
Just some commit message tweaking 😉 Please use:

```
ort-evaluator: Use the correct results file

Use the advisor results file if the scanner is disabled but the advisor is
enabled.
```

(Plus your `Signed-of-by`, of course.)

Reply:
done